ocs_ci/ocs/resources/pod.py

```diff
@@ -1306,6 +1306,12 @@ def wait_for_storage_pods(timeout=200):
     all_pod_obj = get_all_pods(
         namespace=defaults.ROOK_CLUSTER_NAMESPACE
     )
+    # Ignoring pods with "app=rook-ceph-detect-version" app label
+    pods_to_ignore = [
+        pod for pod in all_pod_obj if constants.ROOK_CEPH_DETECT_VERSION_LABEL in pod.get_labels()
+    ]
+    all_pod_obj.remove(pods_to_ignore)
```

Comment:
list.remove() accept a value which is present in the list.

Reply:
good catch